src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -1124,6 +1122,6 @@ external lteq   ::= collapse LTEQ LT EQ       { name = "<=" }
 external oror   ::= collapse OROR OR OR       { name = "||" }
 
 // Trailing commas are allowed
-private meta comma_separated_list ::= <<param>> ( ',' <<param>> ) * ','?
+private meta comma_separated_list ::= <<fn_param>> ( ',' <<fn_param>> ) * ','?
```

Comment:
Hmm and suddenly accessors for arguments of `comma_separated_list` have disappeared when I have tried to fix it.